assembl/static2/js/app/actions/adminActions/voteSession.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+import * as actionTypes from '../actionTypes';
+
+export const updateVoteSessionPageTitle = (locale: string, value: string): actionTypes.UpdateVoteSessionPageTitle => ({
+  locale: locale,
+  value: value,
+  type: actionTypes.UPDATE_VOTE_SESSION_PAGE_TITLE
+});
+
+export const updateVoteSessionPageSubtitle = (locale: string, value: string): actionTypes.UpdateVoteSessionPageSubtitle => ({
+  locale: locale,
+  value: value,
+  type: actionTypes.UPDATE_VOTE_SESSION_PAGE_SUBTITLE
+});
+
+export const updateVoteSessionPageInstructionsTitle = (
+  locale: string,
+  value: string
+): actionTypes.UpdateVoteSessionPageInstructionsTitle => ({
+  locale: locale,
+  value: value,
+  type: actionTypes.UPDATE_VOTE_SESSION_PAGE_INSTRUCTIONS_TITLE
+});
+
+export const updateVoteSessionPageInstructionsContent = (
+  locale: string,
+  value: string
+): actionTypes.UpdateVoteSessionPageInstructionsContent => ({
+  locale: locale,
+  value: value,
+  type: actionTypes.UPDATE_VOTE_SESSION_PAGE_INSTRUCTIONS_CONTENT
+});
+
+export const updateVoteSessionPagePropositionsTitle = (
+  locale: string,
+  value: string
+): actionTypes.UpdateVoteSessionPagePropositionsTitle => ({
+  locale: locale,
+  value: value,
+  type: actionTypes.UPDATE_VOTE_SESSION_PAGE_PROPOSITIONS_TITLE
+});
+
+export const updateVoteSessionHeaderImage = (value: File): actionTypes.UpdateVoteSessionHeaderImage => ({
+  value: value,
+  type: actionTypes.UPDATE_VOTE_SESSION_PAGE_IMAGE
+});
+
+export const updateVoteSessionPage = (value: Object): actionTypes.UpdateVoteSessionPage => ({
```

Comment:
it would be nice to use the type generated from graphql query here (same in legal notice)